backend/tests/conftest.py

```diff
@@ -353,6 +353,22 @@ def delete(self, url, *, params=None, headers=None, cookies=None, auth=USE_CLIEN
         )
 
 
+@pytest.fixture
+def unauthorized_testuser(dbsession, testuser) -> User:
+    # The testuser is a superuser. Remove permissions.
+    dbsession.query(Permission).filter(Permission.recipient_type == RecipientType.USER,
+                                       Permission.recipient_id == testuser.id).delete()
+    return testuser
+
+
+@pytest.fixture
+def unauthorized_testuser(dbsession, testuser) -> User:
+    # The testuser is a superuser. Remove permissions.
+    dbsession.query(Permission).filter(Permission.recipient_type == RecipientType.USER,
+                                       Permission.recipient_id == testuser.id).delete()
+    return testuser
+
+
```

Comment:
Why is this here twice, i.e. doubled?

Looks identical to me…

Oversight, or am I missing something?

Reply:
It should suffice to say: Error between keyboard and chair. Let me delete the duplication...